CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## [6.10.3](https://github.com/algolia/react-instantsearch/compare/v6.10.2...v6.10.3) (2021-03-03)
+
+
```

Comment:
### Bug Fixes

* **RelevantSort:** Rename `SmartSort` widget to `RelevantSort` ([#3026](https://github.com/algolia/react-instantsearch/issues/3026))